wordpress-mobile/WordPress-iOS
# Stats Traffic: Create Today card

Fixes #22393

This is a card that should show 4 values. We already have such a card to show "All-Time" and "Today" views on Insights. According to designs (IqhXWz3Iir7RMb5XH5gGfZ-fi-108%3A4611) we should in fact have similar looking cards for both Insights and Traffic tabs

<img width="312" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/51a5946a-ed0c-4fe3-8288-7b6a03748016">

### Other similar cards

There's also Today Stats Dashboard card, that relies on complete different implementation. As Ravi pointed out (https://github.com/wordpress-mobile/WordPress-iOS/issues/22393#issuecomment-1892964654) it would make sense to update it as well. Created a separate issue for it https://github.com/orgs/wordpress-mobile/projects/222/views/7?pane=issue&itemId=50791091
<img width="304" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/172a91f5-8b75-433f-8576-80cdc6773bf3">

## Solution

Instead of creating new card, update styles and spacing of `StatsTwoColumnRow` and `TwoColumnCell` which uses it to correspond to updates designs.

We can then use `TwoColumnCell` as a Today card in `StatsTraffic` view. We already have `StatsSummaryData` struct that will contain all the information we need.

## To test:

1. Open designs (IqhXWz3Iir7RMb5XH5gGfZ-fi-108%3A4611)
2. Open Jetpack app
3. Stats
4. Insights tab
5. Open Insights management and add all the cards
6. Compare `All Time` and `Today` cards with designs, this will be the look of `Today` card on Stats Traffic tab as well
7. Switch light/dark mode, increase font size, change orientation
8. Observe other cards as well

## Regression Notes
1. Potential unintended areas of impact

Breaking already created cards visually

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual testing

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:


UI Changes testing checklist:
